### ISSUE TITLE
Electoral System

### ISSUE THREAD

**Issue by meatballs** [electora-vote/electora-app#39]
Should Imogen define the electoral system to be used at the time the ballot is created or should that be entirely within Rachel's remit?

e.g. The [UKPA Trustee Election Policy](https://ukpa-internaldocs.readthedocs.io/en/latest/policies/trustee-ballot-policy/) specifies that [Sequential Proportional Approval Voting](https://en.wikipedia.org/wiki/Sequential_proportional_approval_voting) is to be used in some circumstances.

**Comment by meatballs** [comment]
It feels to me that Imogen should specify it so that it's on chain and any count by any Rachel can be verified to be not only accurate but also to have applied the correct system.

**Comment by meatballs** [comment]
Thinking about it further, Imogen has to define the system or else nobody can tell which votes are valid because some systems allow you to select multiple options and others do not.

**Comment by theref** [comment]
yes, the Initiator must specify it - but this is waaay down the line.

**Comment by meatballs** [comment]
Is it? Why?

Why can't we add that to the contract now?

**Comment by theref** [comment]
Oh yeah sure, you can add a placeholder variable
